src/panel-topbar.tsx

```diff
@@ -92,7 +92,7 @@ class PSHeader extends preact.Component<{style: {}}> {
 		if (toRoomList === 'leftRoomList' && toIndex === 0) toIndex = 1; // Home is always leftmost
 		if (toRoomList === 'rightRoomList' && toIndex === PS.rightRoomList.length - 1) toIndex--; // Rooms is always rightmost
 
-		PS[fromRoomList].splice(fromIndex, 1);
+		PS[fromRoomList as 'leftRoomList' | 'rightRoomList' | 'miniRoomList'].splice(fromIndex, 1);
```

Comment:
Am surprised this cast is necessary.

Reply:
```suggestion
		PS[fromRoomList].splice(fromIndex, 1);
```

I checked. It's not.